### ISSUE TITLE
Dump/Restore error for Azure MySQL Databases

### ISSUE THREAD

**Issue by glueeo** [dbeaver/dbeaver#7617]
I have access to my mysql azure account, i could create databases/users ....
But it's impossible to make a dump of any database (nor restore).
However UseSSL is checked, the CA certificate field is fill with the BaltimoreCyberTrustRoot.crt.pem from azure doc 

Here is the error logs from dbeaver (6.3.2)
Task 'MySQL dump' started at Mon Jan 13 20:37:00 CET 2020
-- Connecting to xxx.mysql.database.azure.com...
mysqldump: Got error: 9002: SSL connection is required. Please specify SSL options and retry. when trying to connect
Task 'MySQL dump' finished at Mon Jan 13 20:37:00 CET 2020
2020-01-13 20:37:00.865 - IO error: Process failed (exit code = 2). See error log.
2020-01-13 20:37:00.865 - java.io.IOException: Process failed (exit code = 2). See error log.
	at org.jkiss.dbeaver.tasks.nativetool.AbstractNativeToolHandler.executeProcess(AbstractNativeToolHandler.java:188)
	at org.jkiss.dbeaver.tasks.nativetool.AbstractNativeToolHandler.doExecute(AbstractNativeToolHandler.java:243)
	at org.jkiss.dbeaver.tasks.nativetool.AbstractNativeToolHandler.lambda$0(AbstractNativeToolHandler.java:52)
	at org.jkiss.dbeaver.runtime.RunnableContextDelegate.lambda$0(RunnableContextDelegate.java:39)
	at org.eclipse.jface.operation.ModalContext$ModalContextThread.run(ModalContext.java:122)

Here is the config:
DBEAVER 6.3.2 / Local Client : MySQL Binaries  
Azure Mysql 8+ (with active "Enforce SSL connection")
No Proxy/No tunnel

**Comment by agrisvv** [comment]
today i also checked what dumps (not checked import) are passed via plaintext, so passwords are forwarder in plaintext..
```` 
/usr/bin/mysqldump --routines --add-drop-table --disable-keys --extended-insert --host=xxxx.rds.amazonaws.com --port=3306 -u admin dbname

**Comment by musahaidari** [comment]
It seems that there is no way to set any extra flag for mysqldump in DBeaver

**Comment by Dominic-Preap** [comment]
I have a problem as well with Cloud SQL from Google that when use export data for MySQL (Database -> Tools -> Dump Database). It cannot be achieved but working fine in MySQL Workbench and when I inspected I see there are more flag `--ssl-ca, --ssl-cert, --ssl-key` but they're not present in DBeaver.

**Comment by uslss** [comment]
more flags could be added to highlited fields.
![image](https://user-images.githubusercontent.com/31996417/95377920-dba08900-08eb-11eb-8451-eb6be00a3fb7.png)
Does it solve your problem?

**Comment by Dominic-Preap** [comment]
@uslss yes, that would do the trick but I have to add it a whole lot of string `--ssl-ca 'a/very/long/directory' --ssl-cert 'a/very/long/directory' --ssl-key 'a/very/long/directory'`

It would be nice if Dump/Export can catch this args when to the "Connection Settings" already has the SSL setup

<img width="507" alt="Capture" src="https://user-images.githubusercontent.com/14802170/95406638-11be2700-0945-11eb-846d-44f99a6c9107.PNG">

**Comment by uslss** [comment]
will be implemented in the scope of #12825
